forem/forem
# resolve storybook upgrades

## What type of PR is this? (check all applicable)

- [X] Manual dependency update

## Description

We previously bumped the Storybook versions together in https://github.com/forem/forem/pull/15533, but I had to revert it because running `yarn storybook` locally was throwing errors (see https://github.com/forem/forem/pull/15622).

I tried upgrading using the storybook helper `npx sb@latest upgrade` and also manually upgrading each individual dependency. In both cases I was seeing the same behaviour but couldn't reproduce the error in a fresh storybook repo. There were some dependencies being pulled into the fresh repo's yarn.lock that were suspiciously missing in ours, and the TL;DR is that I think the upgrade path was incorrectly wiping some needed dependencies. I'm curious exactly where this was happening, but I'm not sure we need to spend any more time on it 😅  Removing the packages completely and fresh installing them seems to have fixed the issue.

## Related Tickets & Documents

https://github.com/forem/forem/pull/15622

## QA Instructions, Screenshots, Recordings

- Pull down the code and run `yarn install`
- Run `yarn storybook` and make sure storybook runs and opens in the browser

### UI accessibility concerns?

None, and this upgrade fixes some issues in the storybook accessibility tab

## Added/updated tests?

- [X] No, and this is why: storybook running is test enough

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![David Tennant saying 'bit weird' and making a face](https://media.giphy.com/media/67skGz7YlLW9C229Yf/giphy.gif)
